src/etl/ETLSource.cpp

```diff
@@ -462,7 +464,7 @@ ETLSourceImpl<Derived>::onRead(boost::beast::error_code ec, size_t size)
 
         log_.trace() << "calling async_read - " << toString();
         derived().ws().async_read(
-            readBuffer_, [this](auto ec, size_t size) { onRead(ec, size); });
+            readBuffer_, [this](auto e, size_t _size) { onRead(e, _size); });
```

Comment:
since the parameter size is not used, 
we can 
`ETLSourceImpl<Derived>::onRead(
    boost::beast::error_code ec,
    [[maybe_unused]] size_t _)`

then no need to amend lambda

Reply:
or maybe even `ETLSourceImpl<Derived>::onRead( boost::beast::error_code ec, size_t)`

Reply:
Got primary expression errors when attempting to implement either of your suggestions, so will investigate further into this today

![image](https://user-images.githubusercontent.com/47958858/210513980-a4cc702a-aa56-4113-a5ec-ab06f5168115.png)

Reply:
Attempting either fix leaves invalid conversion and expected primary expressions within parameters due to explicit type definition. 

![image](https://user-images.githubusercontent.com/47958858/210525548-2e70c5fa-1dad-4263-8674-de5db31f5033.png)

Experimented with a few variations: just size_t, size_t _, and _, but all gave compilation errors. 
![image](https://user-images.githubusercontent.com/47958858/210525738-ccaf446c-11a2-4c3b-9277-6dff8873c1ad.png)
![image](https://user-images.githubusercontent.com/47958858/210525914-b89f2e8b-f8a5-4f56-ad17-5317231071d5.png)
![image](https://user-images.githubusercontent.com/47958858/210526278-a867d0d9-896c-4d46-9006-ac4773ba63ca.png)